core/src/main/scala/cats/data/Const.scala

```diff
@@ -39,6 +41,7 @@ final case class Const[A, B](getConst: A) {
   def combine(that: Const[A, B])(implicit A: Semigroup[A]): Const[A, B] =
     Const(A.combine(getConst, that.getConst))
 
+  @nowarn("cat=unused")
   def traverse[F[_], C](f: B => F[C])(implicit F: Applicative[F]): F[Const[A, C]] =
     F.pure(retag[C])
```

Comment:
Just a hint (feel free to disregard): with help of [scalac-compat](https://github.com/typelevel/scalac-compat) it can be addressed in a nicer way:
```
import org.typelevel.scalaccompat.annotation._
```
then
```suggestion
  def traverse[F[_], C](@unused f: B => F[C])(implicit F: Applicative[F]): F[Const[A, C]] =
    F.pure(retag[C])
```

Reply:
```scala
[error] /Users/georgy/work/typelevel/cats/core/src/main/scala/cats/data/Const.scala:26:12: object typelevel is not a member of package org
[error] import org.typelevel.scalaccompat.annotation._
[error]            ^
[error] /Users/georgy/work/typelevel/cats/core/src/main/scala/cats/data/Const.scala:46:26: not found: type unused
[error]   def traverse[F[_], C](@unused f: B => F[C])(implicit F: Applicative[F]): F[Const[A, C]] =
[error]         
```

Reply:
Yeah, I guess it should work better once these changes (typelevel/sbt-typelevel#518) will be available in the current version of the plugin 🤷 

Meanwhile you could add it to `build.sbt` temporarily:
```sbt
ThisBuild / libraryDependencies += "org.typelevel" %% "scalac-compat-annotation" % "0.1.0" % CompileTime
```
... or skip my initial comment about it entirely ...

Reply:
We have a lot of unused (and other warnings in the repo). I guess it would be good to tackle them together.